src/main/java/de/retest/web/WebElementPeer.java

```diff
@@ -121,6 +124,10 @@ private boolean isDefault( final String attributeValue ) {
 		if ( attributeValue.equals( "normal" ) ) {
 			return true;
 		}
+		if ( AttributesProvider.getInstance().getNumericalAttributes().contains( attribute )
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this if-then-else statement by a single return statement. ([squid:S1126](https://sonarcloud.io/organizations/retest/rules#rule_key=squid%3AS1126))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=de.retest%3Arecheck-web&pullRequest=33&issues=AWSQExyEMDLKieESCadp&open=AWSQExyEMDLKieESCadp)

[ISSUE_KEY]: AWSQExyEMDLKieESCadp